style.css

```diff
@@ -0,0 +1,826 @@
+
+/* global */
+
+.hide {
+  display: none;
+}
+
+body {
+  max-width: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+main {
+  padding: 0 7px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body section {
+  margin-bottom: 2rem;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+ul {
+  list-style: none;
+}
+
+.jc-space-between {
+  justify-content: space-between;
+}
+
+.align-items-center {
+  align-items: center;
+}
+
+.align-items-end {
+  align-items: flex-end;
+}
+
+.white-color {
+  color: white;
+}
+
+.gray-color {
+  color: rgb(145, 145, 145);
+}
+
+.orange-color {
+  color: #f42;
+}
+
+.big-section-container {
+  padding: 0 1rem;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.h3-headline {
+  font-size: 1.5rem;
+  font-weight: 700;
+  color: #f42;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Delete all your `color: #f42;` properties in entire CSS file, because you already have a rule, line #57
- [ ] Please use it tin the HTML file

**Example:**

**Html file**

```HTML
<div class="orange">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
color: #f42 has been "DRY'ed"